testbed/lib/keyboard_test_page.dart

```diff
@@ -74,23 +79,29 @@ class _KeyboardTestPageState extends State<KeyboardTestPage> {
     }
 
     int keyCode;
+    String logicalKey;
+    String physicalKey;
     switch (event.data.runtimeType) {
       case RawKeyEventDataMacOs:
         final RawKeyEventDataMacOs data = event.data;
         keyCode = data.keyCode;
+        logicalKey = data.logicalKey.debugName;
+        physicalKey = data.physicalKey.debugName;
         break;
       // The Windows and Linux shells are still sending Android-encoded events.
```

Comment:
This comment needs an update (explaining that Windows uses Linux currently, and briefly why).

Reply:
Done